BUILDING.md

```diff
@@ -1,10 +1,62 @@
 # Warp 10 Setup & Build  
 
+All commands examples below are valid for latest ubuntu or other debian based release.
+
+## Install Java 8 jdk
+```bash
+sudo apt install openjdk-8-jdk
+```
+
 ## Install Thrift
 You need Thrift in order to build the Warp 10 platform.
-Currently, Warp 10 use Thrift release 0.11.0.
+Currently, Warp 10 uses Thrift 0.11.0, to be compatible with some Hbase libs. 
+All major distributions now ships Thrift 0.13+, so you need to compile Thrift.
+
+Make sure Thrift is not already installed:
+```bash
+sudo apt remove thrift-compiler
```

Comment:
This is not needed, simply compile thrift with a specific target via --prefix=...